qlog/src/events/quic.rs

```diff
@@ -743,8 +743,6 @@ pub struct MetricsUpdated {
     pub congestion_window: Option<u64>,
     pub bytes_in_flight: Option<u64>,
 
-    pub ssthresh: Option<u64>,
```

Comment:
This field is still defined in the qlog RFC, so its unclear why its removed from here; see https://quicwg.org/qlog/draft-ietf-quic-qlog-quic-events.html#name-metrics_updated

Reply:
Good point

Reply:
I guess for BBR that would always be 0?

Reply:
None will do

Reply:
Fixed